core/asgi.py

```diff
@@ -11,6 +11,6 @@
 
 from django.core.asgi import get_asgi_application
 
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'kodilancom.settings')
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'core.settings')
```

Comment:
ana paketin adini neden degistirme ihtiyaci duydun?

Reply:
kafa karışıklığını gidermek için

Reply:
Soyle bir sey var, bu uygulamayi `setup.py` ile yayinlarken ederken bu `core` ismi problem olabilir, varilan python paketleri ile cakisabilir. `kodilancom` olarak kalmasinda bir sikinti yok bence ve `pip install appXXXX.whl` ile yukledigimizde site-packages icerisinde `kodilancom` seklinde bir modulumuz olmus olur. Bence `kodilan` app `ilanlar` ya da baska bir isimle degistirilebilir. `core` oldukta generic bir isim.

Reply:
ben bi sıkıntı yaşamadım core konusunda bu tarz giden isimlendiren bir çok yapıda gördüm aslında main app'ı kodilan yapıp diğer app'ı api yada post yapabiliriz.